.github/pull_request_template.md

```diff
@@ -2,19 +2,30 @@
 
 Please include a brief description of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
-Fixes # (issue)
+Fixes #(issue)
 
 ## Type of change
 
 <!--Please delete options that are not relevant.-->
+<!--Type 'x' to mark relevant boxes-->
 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
+## Live Gh-Pages Link
+
+(Upload link here)
```

Comment:
Please refer to GitHub markdown guide and see how hyperlinks are implemented in markdown.

Reply:
sir I am not sure why we are implementing hyperlink here, since the ones who file the PR will themselves upload their gh pages link.

Reply:
The PR's and the comments are written in markdown format on GitHub. That's the way to write hyperlinks on GitHub. `[Live Link](provide link here)` is the way it should be.